### ISSUE TITLE
Bed is always heated after upgrade FW to v0.11.0-241-gffb5105b

### ISSUE THREAD

**Issue by FauthD** [makerbase-mks/MKS-Monster8#40]
Hi, 
with FW mks_monster8 v0.10.0-557.bin my Sermon D1 worked just fine regarding the bed.
Since the neopixel did not work properly (looks like a timing issue), I tried the latest FW compiled locally.
Did a make menuconfig and double checked the values agains the readme.md.
Compiled with `make clean` followed by `make`.
Then flashed with 
`make flash FLASH_DEVICE=/dev/serial/by-id/usb-Klipper_stm32f407xx_39003E001250304B56333920-if00`
(Stopped the klipper service for this).

Now the bed gets heated right after reboot until the emergency shutdown occurs.

Do you know about any changes that could cause this?

**Comment by FauthD** [comment]
Look like this is the same issue as this:
[V2 board starts burning when attempting dfu mode #26](https://github.com/makerbase-mks/MKS-Monster8/issues/26).

I measured the Q7 turns on the heater, voltage at R38/R40 junction is zero. The Raspberry was not connected for this test. Q7 gets very hot.

If MKS reads here, can you provide the Q7 and tell which are the correct values for the resistors please?

**Comment by MKS-hosiey** [comment]
R38 100 Ω，R40 100kΩ，Q7 

> Look like this is the same issue as this: [V2 board starts burning when attempting dfu mode #26](https://github.com/makerbase-mks/MKS-Monster8/issues/26).
> 
> I measured the Q7 turns on the heater, voltage at R38/R40 junction is zero. The Raspberry was not connected for this test. Q7 gets very hot.
> 
> If MKS reads here, can you provide the Q7 and tell which are the correct values for the resistors please?
R38 100 Ω，R40 100kΩ，Q7  HYG024N03LR1D  TO-252

**Comment by FauthD** [comment]
Hi, 
The official MKS-Shop at Aliexpress provided me with the correct transitor (I had to pay 1.08€). Thanks!
Replacing it was not too bad with my cheap hot-air station. Better would be some heating underneat the board, but it worked out well.
Now the board works fine again.

Trying to find the root cause did not expose something:

1. Disconnect the bed heater just in case.
2. Flash the original FW from this github page. (V0.10.x)
3. Reboot
4. Update to latets FW with "make flash ...".
5. Measure the gate voltage of Q7 with Oscilloscope. It was zero all the time, so the Q7 should not have been turned on (or partially turned on as I suspected).
6. Do the same again with the latest FW (V0.11.x), same result.
7. If bed needs heat, the Gate voltage is 3.3V.

So I cannot explain the issue, though I do not believe it was pure coincidence.
I recommend to disconnect the heater bed while DFU, or just use the USB supply.

PS: I think R40 is only 10K, not 100K as MKS-hosiey mentioned.

Regards, Dieter Fauth
